cpp/oneapi/dal/backend/primitives/selection/BUILD

```diff
@@ -0,0 +1,32 @@
+package(default_visibility = ["//visibility:public"])
+load("@onedal//dev/bazel:dal.bzl",
+    "dal_module",
+    "dal_test_suite",
+)
+
+dal_module(
+    name = "selection",
```

Comment:
Please add dependency on `selection` to `cpp/oneapi/dal/backend/primitives/BUILD`, see `collect_modules` and `collect_test_suites`.